src/main/java/at/porscheinformatik/sonarqube/licensecheck/gradle/GradleDependencyScanner.java

```diff
@@ -28,6 +28,9 @@
 
 public class GradleDependencyScanner implements Scanner
 {
+    public static final String JSON_REPORT_PATH_PROPERTY = "sonar.licenseCheck.jsonReportPath";
```

Comment:
- Please move the configuration to `LicenseCheckPropertyKeys`
- The config should also be registered in the plugin like this https://github.com/porscheinformatik/sonarqube-licensecheck/blob/2671581a0e6a6188882075b594c371148621e7be/src/main/java/at/porscheinformatik/sonarqube/licensecheck/LicenseCheckPlugin.java#L130
- Please rename the config to "licensecheck.gradle-json-report-path"